CONTRIBUTE.md

```diff
@@ -54,49 +55,95 @@
     └── sphinx_airflow_theme
 ```
 
-## Working with the project
+# Working with the project
+
+Work with the site and documentation requires that your computer be properly prepared. Most tasks can
+be done by the site.sh script
+
+### Prerequisite Tasks
```

Comment:
Mentioning pre-commit would be helpful for people not used to it.
```sh
brew install pre-commit
pre-commit install
```

Reply:
Thanks. I added section.